packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The `nimble-table-column-mapping` is a component that supports rendering specific number, boolean, or string values as mapped text. `nimble-table-column-icon` is a specialized version of `nimble-table-column-mapping` that instead maps values to icons and/or spinners and has a minimal, fixed width. The actual mappings are defined by child elements `nimble-mapping-icon`, `nimble-mapping-spinner`, and `nimble-mapping-text`.
+The `nimble-table-column-enum-text` is a component that supports rendering specific number, boolean, or string values as mapped text. `nimble-table-column-icon` is a specialized version of `nimble-table-column-enum-text` that instead maps values to icons and/or spinners and has a minimal, fixed width. The actual mappings are defined by child elements `nimble-mapping-icon`, `nimble-mapping-spinner`, and `nimble-mapping-text`.
```

Comment:
Is "`nimble-table-column-icon` is a specialized version of `nimble-table-column-enum-text`..." accurate, or should it be `nimble-table-column-icon` is a specialized version of `TableColumnEnumBase`?

Reply:
Removed that bit about it being a specialized version of something else.